website/docs/components/form/toggle/partials/accessibility/accessibility.md

```diff
@@ -1,9 +1,11 @@
-#### Known Issues
+## Conformance rating
```

Comment:
unsure if we have conformance ratings for other variations of this component or not

Reply:
@MelSumner do you know?